### DISCUSSION TITLE
Custom lists of objects

### DISCUSSION THREAD

**Discussion by PeterBaker0** [twentyhq/twenty#8231] [Ideas]
## Scope & Context

We would like to be able to create lists of objects without having to create a view which filters over a value. 

## Current behavior

Say I want a list which is a saved subset of People. Right now, I have to create some new property over which I can filter, so that this list reflects my selection of People. 

## Expected behavior

I would like a new type of View which is just a List. This list has a selection of People (or any object type), and can be added to and removed from. This List is shared between users, and in all other ways operates the same as other Views. The List could be filtered, but this filter would not be saved to the List (since the list inclusion/exclusion itself is the filter). The List could be ordered, and this order would be shared between users. 

**Example:** 

I want to create a 'campaign email list' which is just a selection of People. I can then export from this list easily, for example. Right now it takes a lot of steps to do this very simple action - i.e. data model-> add property -> add view  -> filter by this property -> go back to all people -> modify property -> see this reflected in filtered list. I would prefer the ability to just maintain standalone lists.

**Comment by Bonapara** [comment]
Thanks for the suggestion, turning this into a discussion!

**Comment by PeterBaker0** [comment]
@Bonapara no worries! It would be a great feature for us.

**Comment by Bonapara** [comment]
Favorites and favorites folders are about to be released. You won't be able to see their content in lists or kanbans yet, but they will eventually become a list feature.

**Comment by PeterBaker0** [comment]
@Bonapara We had to work around this today in an operational deployment. We want to track participants in user testing rounds. To do this, it required 

- add a user research multi select
- add an option for each round (e.g. round 1, round 2 etc)
- add a filtered list view on the multi select values
- add people by checking that option

Then we wanted to track properties about that round of user testing which might only apply in that context - for example a field might be a boolean 'round 1 prep completed' or 'round 2 ethics consent signed'. Currently to add these fields we would need to either use notes (which can't be tabulated well), or add fields to every Person, even though only a very small subset would use the field. 

I think this would be a killer feature to a) allow lists of a specific object which are self-contained requiring no additional field to manage (or at least this is hidden from the user) b) allow attributes/fields to be specific to a list, so that property is only visible/usable for that object if they are involved in that list.

Then the workflow would become

- create a list e.g. 'round 1 user testing'
- add people to the list
- add property to list (ethics signed)

Thanks!

**Reply by Bonapara** [reply]
Thanks for sharing this use case @PeterBaker0

**Reply by PeterBaker0** [reply]
No worries
